crates/rpc/rpc-api/src/bundle.rs

```diff
@@ -26,8 +39,8 @@ pub trait EthBundleApi {
     #[method(name = "callBundle")]
     async fn call_bundle(
```

Comment:
can you remove this fn and make `trait EthBundleApi: EthCallBundleApi`? maybe not possible

Reply:
possible but not really useful here imo